Wrappers/Python/cil/optimisation/operators/Operator.py

```diff
@@ -492,7 +515,16 @@ def calculate_norm(self):
 
 
 class CompositionOperator(Operator):
+    """Composes one or more operators. 
+    For example, `CompositionOperator(left, right).direct(x)` is equivalent to `left.direct(right.direct(x))`
 
+    
+    Parameters
+    ----------
+    args: `Operator`s
+        Operators to be composed. As in mathematical notation, the operators will be applied right to left
+    

```

Comment:
Please add docstring explaining 'out' parameter in direct

Reply:
Would be great if this can be done for SumOperator as well

Reply:
Thanks @lauramurgatroyd